### PR TITLE
docs(readme): explain the optional setup when developing locally

### PR DESCRIPTION
**Pull request by tai271828**
## Types of changes

- **Documentation Update**

## Description

Update README to explain the optional setup when developing locally

## Checklist:
- [ ] Add test cases to all the changes you introduce
- [ ] Run `poetry run pytest` locally to ensure all linter checks pass
- [ ] Update the documentation if necessary

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -66,4 +68,4 @@ rows = query_job.result()  # Waits for query to finish
 
 for row in rows:
     print(row.diet)
-```
\ No newline at end of file
+```
```
Comment (tai271828) [inline README.md:71]: fix `\ No newline at end of file`

**Review by david30907d — APPROVED** [review]
LGTM, thanks for updating the readme~

**Comment by tai271828** [comment]
CI failed for the other (and previous) cause. May @david30907d  merge this PR if it looks good to you?

**Comment by david30907d** [comment]
@tai271828 I knew why we cannot pass the CI. You created this PR from your repo, so it cannot access the secrets that I had set up in `pycon-etl` repo 😅 

seems like you're using `github flow` and I'm using `gitlab flow`

I'm okay with either, just wondering how to setup the secrets in `github flow`?
Otherwise `github flow` seems more appropriate for open source project haha

**Comment by tai271828** [comment]
This pull request is out-dated for previous main branch force-update. Close it.

**Pull request closed without merging**